server/restafary.js

```diff
@@ -190,11 +190,10 @@ async function sendArchiveListing(path, params) {
     try {
         let data = await archive.listArchive(path);
         let header = `Archive contents for ${path}`;
-        header += `\n${"=".repeat(header.length)}`;
+        header += `\n${'='.repeat(header.length)}`;
         data = `${header}\n${data}`;
         ponse.send(data, params);
-    } catch (ex) {
-        console.log(ex);
+    } catch {
         ponse.sendError('Unable to read archive contents for ' + path, params);
```

Comment:
Let's move check into `listArchive`, and change it's signature into:

```js
const [error, list] = await listArchive(past);
```

Also let's create data in `archive.js` file, and here only send error or data.